hack/install-vm.sh

```diff
@@ -141,26 +149,261 @@ EOF
     systemctl status "$ANTREA_AGENT"
 }
 
+# Generic function to execute command with timeout
+execute_command() {
+    cmd="$1"
+    timeout="$2"
+    echo "Executing $cmd with timeout $timeout at $(date)"
+    timeout --preserve-status --foreground $timeout $cmd
+    ret=$?
+    if [ $ret -ne 0 ]; then
+        echoerr "$cmd failed with status $ret at $(date)"
+        echoerr "Exiting!! please check internet connectivity and re-run install script"
+        exit 2
+    fi
+}
+
+# Install docker as container runtime
+install_container_runtime() {
+    os=`grep -Po '^ID=\K.*' /etc/os-release | sed -e 's/^"//' -e 's/"$//'`
+    echo "Installing container runtime on $os"
+    case "$os" in
+        ubuntu)
+        # Uninstall older versions of Docker(called docker, docker.io, or docker-engine)
+        docker_uninstall_packages="docker docker-engine docker.io containerd runc"
+        docker_install_packages="ca-certificates curl gnupg lsb-release"
```

Comment:
rename it, since it is not docker packages.

```suggestion
        docker_prereq_packages="ca-certificates curl gnupg lsb-release"
```